src/app_handle.rs

```diff
@@ -279,6 +279,19 @@ impl ApplicationHandle {
                     window_builder = window_builder.with_decorations(false);
                 }
             }
+            if let Some(undecorated) = config.undecorated {
+                window_builder = window_builder.with_decorations(!undecorated);
```

Comment:
unnecessary double negation, this should not be `!` and called `undecorated`

Reply:
> Every toolkit I've worked with for years has referred to the special-case of windows with no frame or title-bar as "undecorated".

We're not every toolkit and how lack of decorations is referred to doesn't have to dictate how API looks like.

> Consider that if we called this property "decorated" and had a method with that name, what the person trying to call the method is going to perceive is that undecorated windows are the default (they aren't and shouldn't be), and they must call decorated(true) to avoid that.

By that logic `show_titlebar` or other options are also `false` by default.
And while being on topic of `show_titlebar`, I find the `undecorated` option to be redundant and harmful given that we already configure decorations in the titlebar option. This should be adjusted or consolidated.

Reply:
*Undecorated* means something different.

On most desktop OS's, a window that had its titlebar hidden still has a system-drawn *frame* that may or may not allow the window to be moved or resized.

*Undecorated* means something different - the application is responsible for painting the **entire** visual representation of the window, and the system paints nothing at all.

It is a genuinely different thing than hiding the titlebar, and has been used for things like popups and shaped, non-rectangular windows since the 90s.

Make just one - either one - available, and there are some applications you should be able to write with Floem but you can't.

Reply:
And, show_titlebar **should** be hide_titlebar - the caller has expectations of what *make me a window* means, trained by decades of desktop UI APIs, and the API should express the way you can ask that the window *deviate* from the platform default.

But I'm trying to flesh out missing functionality, not make gratuitous breaking changes in an existing API.